portfolio/src/main/webapp/WEB-INF/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Is this file necessary since you expanded the other one?

Reply:
Not sure how I ended up having two. Deleted the old one though.